apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -159,6 +162,15 @@ defmodule AndiWeb.EditLiveView do
     |> complete_validation(socket)
   end
 
+  def handle_event("validate", %{"form_data" => form_data, "finalize_form_data" => finalize_form_data}, socket) do
+    socket = assign(socket, :finalize_form_data, finalize_form_data)
```

Comment:
All of this is largely b/c you can't nest form tags, and thus, can't let the finalize form handle itself

Reply:
I will move this over if you merge before me